### DISCUSSION TITLE
Part of the history notification is missing

### DISCUSSION THREAD

**Discussion by iamxiaojianzheng** [folke/snacks.nvim#106] [Q&A]
This error notification cannot be viewed in Noice

![image](https://github.com/user-attachments/assets/7703b146-e618-4707-80fa-de42f3b5466b)

open this with `<leader>sna`

![image](https://github.com/user-attachments/assets/870c3888-bede-4a45-b2f7-c81ddff5a46a)

**Comment by iamxiaojianzheng** [comment]
I found that after updating to snacks, some notification information was missing when viewing historical notification messages through notice.

**Comment by iamxiaojianzheng** [comment]
In lazyvim12, I can view historical messages and notifications using the commands `message` and `Notification`

**Comment by folke** [comment]
That's because snacks loads before noice takes over `vim.notify`.
Either way, I just added a work-around for this in LazyVim.
